mobile-app/src/components/UserCard.tsx

```diff
@@ -200,35 +197,54 @@ function CommonHabits({ friendId }: { friendId: string }) {
   );
 }
 
-// // Takes an array of userids, currently uses numbers but should be strings
-// function MutualFriends({ mutualFriends }: { mutualFriends: number[] }) {
-//   const mutual= mutualFriends.map((friendId) =>
-//     useAtomValue(friendA(friendId)),
-//   );
-//   const { colorScheme } = useColorScheme();
-//   return (
-//     <View className="mt-1 flex flex-row items-center">
-//       {/* // TODO: Add user pfps once database query for mutuals is implemented */}
-//       {mutualmap((friend) => (
-//         <View
-//           key={friend.id}
-//           className="-mr-[7px] rounded-full border border-stone-400"
-//         >
-//           <SmallProfilePicture picUrl={friend.profilePicUrl} />
-//         </View>
-//       ))}
-//       {mutualFriends.length == 0 ? (
-//         <Text className="ml-3 text-xs font-semibold text-stone-400">
-//           No mutual friends
-//         </Text>
-//       ) : (
-//         <Text className="ml-3 text-xs font-semibold text-stone-400">
-//           {mutualFriends.length} mutual friends
-//         </Text>
```

Comment:
This text that shows the number of mutual friends should always be visible. Right now it only shows up when they have more than the max.

![image](https://github.com/owengretzinger/habits-together/assets/73363563/ac0b78ed-97f0-4eac-9af8-78b07d519964)